src/global/config.ts

```diff
@@ -1,4 +1,26 @@
-export class Config {
+/**
+ * Configuration options for limel-config.
+ *
+ * @public
+ */
+export type Config = {
+    /**
+     * The path to where the icon library used by limel-icon is located.
+     */
+    iconPath?: string;
+
+    /**
+     * The default locale to use for components that support localization.
+     */
+    defaultLocale?: string;
+
+    /**
+     * @internal
+     */
+    featureSwitches?: any;
```

Comment:
This should probably be `Record<string, boolean>`

Reply:
⚡